src/scenes/Login/components/SignUp.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+const SignUp = () => {
```

Comment:
We don't need a signup form, just login are enough

Reply:
Ok sis =]]